.github/workflows/pr_checks.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: PR Checks
```

Comment:
If we include `push` as I mention in a separate comment in this review, we might also want also change this.

`cordova-eslint` currently calls it `name: Node CI` but I am unsure what is best.

Reply:
I used https://github.com/actions/starter-workflows/blob/master/ci/node.js.yml as a template for the workflow of `cordova-eslint`. That's also where the name comes from. I think it's better than PR Checks